File: CodeForces/Java/CF_236A_BoyOrGirl.java
//  Problem: 236A. Boy Or Girl
//  Link: https://codeforces.com/problemset/problem/236/A

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);
    String str = scan.nextLine();
    scan.close();

    StringBuilder unique = new StringBuilder();
    for (int i = 0; i < str.length(); i++) {
      String currentChar = Character.toString(str.charAt(i));
      if (!unique.toString().contains(currentChar)) {
        unique.append(currentChar);
      }
    }

    if (unique.length() % 2 == 0) {
      System.out.println("CHAT WITH HER!");
    } else {
      System.out.println("IGNORE HIM!");
    }
  }
}

File: CodeForces/Python/CF_236A_BoyOrGirl.py
# Problem: 236A. Boy Or Girl
# Link: https://codeforces.com/problemset/problem/236/A

string = input()
unique = []
for i in string:
  if i not in unique:
    unique.append(i)
if len(unique)%2 == 0:
  print("CHAT WITH HER!")
else:
  print("IGNORE HIM!")